common/actions.js

```diff
@@ -98,7 +98,15 @@ export const getSlateById = async (data) => {
 
 export const getSlatesByIds = async (data) => {
   await Websockets.checkWebsocket();
-  return await returnJSON(`/api/slates/get`, {
+  return await returnJSON(`/api/v1/explore`, {
+    ...DEFAULT_OPTIONS,
+    body: JSON.stringify({ data }),
+  });
+};
+
+export const getSocial = async (data) => {
+  await Websockets.checkWebsocket();
+  return await returnJSON(`/api/v1/social`, {
```

Comment:
I think since this is a user related api call, it should go under api/users, not api/v1 (v1 is for external apis I believe, like the ones the chrome extension uses). And maybe name it get-peers or something?

Reply:
I see. for the name maybe get-social? it's fetching peers and following slates